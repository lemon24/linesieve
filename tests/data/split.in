-s '(\S+):$'
split -s space
split -s pattern -d '\s+'
split -s fixedfields -F -d . -f-2
split -s xod -X -d ' \d [a-z] ' -D:
---
space:
 a  b   c
a b c
pattern:
a  b c
 a b
fixedfields:
a.b.c
a b.c
xod:
a2xb1y
