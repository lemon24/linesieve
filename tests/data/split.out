space
a	b	c
a	b	c
pattern
a	b	c
	a	b
fixedfields
a.b
a b.c
xod
a:b:
